### PR TITLE
Update queue to use correct date field (bug 1084654)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mkt/reviewers/tests/test_models.py [hunk 1]
```diff
@@ -32,6 +34,15 @@ def setUp(self):
         self.grant_permission(self.admin_user, '*:*', name='Admins')
         user_factory(email='regular')
 
+        # Disconnect indexing signals on rereview and escalation models.
```
Comment (diox) [inline mkt/reviewers/tests/test_models.py]: why ?

Comment (robhudson) [inline mkt/reviewers/tests/test_models.py]: Ah, let me remove that now. I had a theory those were causing test problems but let me try again without. I did this early on in this branch.

**Comment by robhudson** [comment]
Updated to remove the signal disconnects.

**Comment by diox** [comment]
r+
